cg/services/order_validation_service/rules/utils.py

```diff
@@ -41,3 +40,37 @@ def is_volume_missing(sample: Sample) -> bool:
     if is_in_container(sample.container) and not sample.volume:
         return True
     return False
+
+
+def has_sample_invalid_concentration(sample: SampleWithSkipRC, store: Store) -> bool:
+    application: Application | None = store.get_application_by_tag(sample.application)
+    if not application:
+        return False
+    concentration: float | None = sample.concentration_ng_ul
+    is_cfdna: bool = is_sample_cfdna(sample)
+    allowed_interval: tuple[float, float] = get_concentration_interval(
+        application=application, is_cfdna=is_cfdna
+    )
```

Comment:
There is a bit of code duplication: These lines are also present in `cg/services/order_validation_service/rules/sample/utils.py:create_invalid_concentration_error`. It seems like that function could call this one. Feel free to resolve if this involves too many changes, but maybe we could keep code duplication to a minimum since we know sonar cloud can be a problem

Reply:
Extracted into separate method